src/core/public/keyboard_shortcuts/keyboard_shortcuts_service.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { KeyboardShortcutsSetup, KeyboardShortcutsStart, ShortcutDefinition } from './types';
```

Comment:
nitpick: lets call it (and file names and everything) `KeyboardShortcut`, not `KeyboardShortcuts`